app/src/main/java/com/github/se/project/ui/navigation/NavigationActions.kt

```diff
@@ -25,6 +25,7 @@ object Screen {
   const val EDIT_PROFILE = "Edit profile Screen"
   const val CREATE_TUTOR_PROFILE = "Tutor information creation Screen"
   const val CALENDAR = "Calendar Screen"
+  const val ADD_LESSON = "Add Lesson"
```

Comment:
we might want to name it "Add lesson Screen" to be consistent with previous declarations :)

Reply:
Done